app/packages/dataset/src/index.tsx

```diff
@@ -1,9 +1,9 @@
-import * as fos from "@fiftyone/state";
 import React, { useRef, useState } from "react";
 import { createRoot } from "react-dom/client";
 import { RecoilRoot, useRecoilState } from "recoil";
 import { RecoilRelayEnvironmentProvider } from "recoil-relay";
 import { DatasetRenderer } from "./Dataset";
+import { getEnvProps, fos } from "./";
```

Comment:
is `getEnvProps` used?

Reply:
It was exported at one point - @benjaminpkane might have refactored this.

Reply:
Still exported based on these changes, just not used here.

Reply:
Will get rid of it.